### PR TITLE
Add force_float test (pyvista#3333)

### PR DESCRIPTION
**Pull request by sgaist**
### Overview

This pull request will implement additional tests for the vtk_points helper.

resolves https://github.com/pyvista/pyvista/issues/3333

### CODE REVIEW

File: tests/test_helpers.py [hunk 1]
```diff
@@ -348,4 +348,10 @@ def test_vtk_points_deep_shallow():
     assert not np.array_equal(vtk_points.GetPoint(0), points[0])
     assert np.array_equal(points[0], [0.0, 0.0, 0.0])
 
-    # TODO: test force_float
+
+@pytest.mark.parametrize("force_float,expected_data_type", [(False, np.int64), (True, np.float32)])
+def test_vtk_points_force_float(force_float, expected_data_type):
+    np_points = np.array([[1, 2, 3]], dtype=np.int64)
+    pset = pyvista.PointSet(np_points, force_float=force_float)
```
Comment (MatthewFlamm) [inline tests/test_helpers.py:355]: This test is fine after the intricacies are worked out.  My comment from the previous PR was that we should directly test `vtk_points` to fully close #3333.  For example this line would be `vtk_points = pyvista.vtk_points(np_points, force_float=force_float)` and test that this object has the correct type.

Comment (sgaist) [reply tests/test_helpers.py:355]: By bad, I moved the code and somehow mixed the content of the two files.

It's now fixed.

**Review by akaszynski — DISMISSED** [review]
LGTM. Thanks!

**Review by MatthewFlamm — DISMISSED** [review]
LGTM.  Thanks @sgaist

**Review by banesullivan — APPROVED** [review]

**Comment by sgaist** [comment]
@akaszynski sorry, I did not thought that rebasing the branch would trigger a force push like that. I'll avoid that in the future.

**Pull request merged**